### PR TITLE
revert az aks copilot

### PR DESCRIPTION
**Pull request by sulixu**
---
revert commits related with az aks copilot

https://github.com/Azure/azure-cli-extensions/pull/6629
https://github.com/Azure/azure-cli-extensions/pull/6750

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -16,10 +16,6 @@ Pending
 +++++++
 * Add `--disable-workload-identity` to the `az aks update` command.
 
-0.5.156
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:19]: maybe you should keep this version history as it has been released

**Review by FumingZhang — APPROVED** [review]
LGTM

Please resolve conflicts

**Review by ReaNAiveD — APPROVED** [review]

File: src/aks-preview/HISTORY.rst (hunk 1 above)
Comment (yanzhudd) [reply src/aks-preview/HISTORY.rst:23]: Is there any updates for this comment?

**Comment by yanzhudd** [comment]
It is better to add the link of feature PR in https://github.com/Azure/azure-cli-extensions/pull/6773#issue-1902045965

**Pull request merged**